doc/image-from-registry.md

```diff
@@ -2,47 +2,50 @@
 The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be imported into a PV.
 
 ## Prerequisites
-The VM disk image should be called 'disk.img' and placed in the root folder.
+Import from registry should be able to consume the same container images as [containerDisk](https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md).
+Thus the VM disk image file to be consumed must be located under /disk directory in the container image. The file can be in any of the supported formats : qcow2, raw, archived image file. There are no any special naming contraints for the VM disk file.
```

Comment:
There are no special naming constraints for the VM disk.

Extra 'any' and constraints is misspelled.